### ISSUE TITLE
Batch of records need to be reset to pending

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#838]
A glitch when bulk editing has produced a set of 2,286 records that need to be reset to the "pending verification" state. Based on previous issues of this type I think the following needs to be done (see also #692).

Occurrence IDs are listed in this spreadsheet, based on all occurrences verified by user_id 44065 where verified_on = 23/04/2020  13:19:48

[Matt Smith verification issue.xlsx](https://github.com/BiologicalRecordsCentre/iRecord/files/4579901/Matt.Smith.verification.issue.xlsx)

For each of the records defined in the above query, we need to:
- set o.record_status to 'C'
- set o.verified_by_id to Null
- set o.verified_on to Null
- set o.record_substatus to Null
- set o.record_decision_source to Null
- delete the verification comment associated with this bulk verification (see below)
- set up cache_builder entries for these occurrrences in the work_queue to feed changes through to the cache tables.

Rich's notes on the verification comment were:

> The auto-verifier also inserts records into the occurrence_comments table. These have their created_on field set with the same timestamp used to set the verified_on field of the occurrences table. That means we could find these and set their deleted attribute to true by matching their occurrence_id and created_on fields to the records we are going to modify in the occurrences table before we modify them. 

Marking this as urgent to minimise the risk of incorrectly verified records getting downloaded.

**Comment by burkmarr** [comment]
@kitenetter This is done and jobs to rebuild cache are in the work_queue. The query to select occurrences verified by user_id 44065 at  '23/04/2020 13:19:48' selected one more than in the spreadsheet, but I don't think that's a problem.

Selecting the associated comments to mark as deleted was a little more involved than the auto_verify mentioned above because, for whatever reason, the created_on timestamp did not match the verified_on time in the occurrence table and was spread over a few seconds across the relevant records, but by looking at the records I found I could identify them by using a range. I picked up a handful of comments made on records after the bulk-verification indicating that the bulk verification was incorrect. Since we're aiming to delete all trace of the bulk-verification, I also marked these as deleted.

I'll keep an eye on the work queue and update the issue when they're done.
